apps/client-asset-sg/src/app/components/disclaimer-dialog/disclaimer-dialog.component.scss

```
@@ -1,22 +1,34 @@
+::ng-deep .cdk-overlay-dark-backdrop {
+  background-color: grey;
+}
```

`::ng-deep` **always** needs a prefix that is not inside the `::ng-deep`, as it is otherwise makes everything after it global.
```suggestion
:host ::ng-deep .cdk-overlay-dark-backdrop {
  background-color: grey;
}
```
> Note that `::ng-deep` is [deprecated](https://stackoverflow.com/a/49308475), without adding a replacement. This means there is no other choice but to use it in situations like these, while knowing that it is has some issues and might be removed at some point.